consensus.go

```diff
@@ -0,0 +1,235 @@
+package dcrlibwallet
+
+import (
+	"fmt"
+	"sort"
+	"time"
+
+	"decred.org/dcrwallet/v2/errors"
+	w "decred.org/dcrwallet/v2/wallet"
+	"github.com/decred/dcrd/chaincfg/chainhash"
+	"github.com/decred/dcrd/chaincfg/v3"
+)
+
+// GetVoteChoices returns configured vote preferences for each agenda of the
+// latest supported stake version for the wallet with the specified walletID.
+func (mw *MultiWallet) GetVoteChoices(hash string, walletID int) (*GetVoteChoicesResult, error) {
```

Comment:
This isn't true though, it returns data for all agendas of all stake versions, not just the latest. Admittedly, the vote preferences are currently returned for only agendas of the latest stake version because there's currently no way to get previous agenda vote preferences. Comment needs an update to clarify the data being returned - data for all agendas of all stake versions but only preferences saved for the current stake version agendas.

To this end, I'm not sure this method should continue to be called `GetVoteChoices`, it doesn't really do that. The upstream method of this name only deals with the current stake version and thus is able to return the vote choices correctly, so it can correctly say it is getting vote choices. This seems to me to be getting all agendas (with vote choices for the latest/current agendas) rather than getting vote choices.